system.go

```diff
@@ -28,6 +28,32 @@ type SystemInput struct {
 	Note        *string          `json:"note,omitempty"`
 }
 
+func (s *System) GetTag(tagId ID, client *Client) *Tag {
+	if s == nil {
+		return nil
+	}
+	tags, err := s.SystemId.Tags(client, nil)
+	if err != nil {
+		fmt.Println(fmt.Errorf("Error getting tags: %s", err))
+		return nil
+	}
+	for _, tag := range tags.Nodes {
+		if tag.Id == tagId {
+			return &tag
+		}
+	}
+	fmt.Println(fmt.Errorf("Error getting tags: %s", err))
```

Comment:
the client library shouldn't have any output like this - it should go into an error if its an error otherwise return a sane default